docs/sources/datasources/graphite.md

```diff
@@ -18,32 +18,21 @@ Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instruct
 
 To access Graphite settings, hover your mouse over the **Configuration** (gear) icon, then click **Data Sources**, and then click the Graphite data source.
 
-| Name                  | Description                                                                                                                           |
-| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| `Name`                | The data source name. This is how you refer to the data source in panels and queries.                                                 |
-| `Default`             | Default data source means that it will be pre-selected for new panels.                                                                |
-| `URL`                 | The HTTP protocol, IP, and port of your graphite-web or graphite-api install.                                                         |
-| `Access`              | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser. |
```

Comment:
shouldn't we keep this here until it's actually removed?

Reply:
I've actually removed it. It will now show up only when it's already set (by provisioning on previous settings).

Reply:
saw the changes now, makes sense 👍